.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+* To merge open pull requests into `devel` regularly (see [below](#merging-contributions)).
+* To generate new releases of the model regularly (see [below](#releasing-a-new-version)).
+
+### Merging contributions
+The following points should be considered when merging branches to `devel`:
+* Make sure the branch gets accepted by at least one developer with writing access.
+* Wait at least a day before merging, to allow other developers to inspect the pull request.
+* As soon as the branch is merged, check if `devel` is still possible to merge to `master` (this can be checked [here](https://github.com/SysBioChalmers/yeast-GEM/compare/devel)). If conflicts appear (which should rarely happen and only if the `.xml` file was changed in an unexpected way by a toolbox update), fix the conflict _locally_ as soon as possible in `devel` and then push it (note, **DO NOT** pull any other changes from `master` to `devel`, just the single file that is creating the conflict).
+
+### Releasing a new version
+
+* A merge of `devel` with `master` invokes a new release.
+* A new release should be made as soon as there is substantial new work in `devel` (as rule of thumb, after around 3 pull request merges).
+
+yeast-GEM follows [semantic versioning](https://semver.org/), adapted to GEMs:
+* A `major` release is seldom used and only meant for a new publication. Backwards compatibility should be, ideally, always preserved.
+* A `minor` release involves a substantial change in the model (several new reactions/metabolites/genes), such as:
+  * Addition of genes/reactions/metabolites from a whole genome annotation.
+  * Addition of several annotation fields.
+  * Inclusion of a major new formalism in the model.
+  * Addition of a plurality of pathways.
+* A `patch` release is the most common one and is done when only few things have changed in the model, or there are only changes that have to do with format, such as:
+  * Adding a single new annotation field.
+  * Fixing some chemical formulas/charges.
+  * Updating toolboxes.
+  * Re-organization of data
+  * Refactoring of code.
+  
+When releasing, please follow these steps:
+  1. Make sure all dependencies in `devel` correspond to the setup from the local computer from which the release will be made. If not, make a single commit in `devel` updating this with a `loadYeastModel`/`saveYeastModel` cycle.
+  2. Create a pull request from `devel` to `master`, indicating all new features/fixes/etc. and referencing every previous pull request included. Tip: if any [issue](https://github.com/SysBioChalmers/yeast-GEM/issues) gets solved in the release, write in the pull request description "Fixes #X", where "X" is the issue number. That way the issue will be automatically closed after merge.
+  3. Merge at least a day after (having at least one accepted review).
+  4. Switch locally to `master` and update `history.md`, by putting at the top a description with what is included in the new version.
```

Comment:
Should new entries in `history.md` not have the same text as the PR to master?

Reply:
yes it should actually be the same text, I have clarified that now